docs/variants/protectli_vp66xx/releases.md

```diff
@@ -1,20 +1,82 @@
+- [Support for discrete TPM](https://docs.dasharo.com/unified-test-documentation/dasharo-security/200-tpm-support/)
+- [UEFI Secure Boot support](https://docs.dasharo.com/unified-test-documentation/dasharo-security/206-secure-boot/)
+- [Boot logo customization support](https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality/)
+- [USB boot support](https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/31N-usb-boot/)
+- [NVMe boot support](https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/312-nvme-support/)
+- [TPM Measured Boot](https://docs.dasharo.com/unified-test-documentation/dasharo-security/203-measured-boot/)
+- [UEFI Shell](https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/30P-uefi-shell/)
+- [UEFI Secure Boot](https://docs.dasharo.com/unified-test-documentation/dasharo-security/206-secure-boot/)
+- [Network boot](https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/315b-netboot-utilities/)
+- [Windows 11 booting](https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/31A-windows-booting/)
+- [Ubuntu LTS booting](https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/308-debian-stable-and-ubuntu-lts-support/)
+- [Serial port console redirection](https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio/#sio004001-serial-port-in-firmware)
+- [Vboot Verified Boot](https://docs.dasharo.com/guides/vboot-signing/)
+- [Intel ME HAP disable](https://docs.dasharo.com/unified-test-documentation/dasharo-security/20F-me-neuter/)
+- [BIOS flash protection for Vboot recovery region](https://docs.dasharo.com/unified-test-documentation/dasharo-security/20J-bios-lock-support/)
+- [Setup menu password configuration](https://docs.dasharo.com/dasharo-menu-docs/overview/#dasharo-menu-guides)
+- [SMM BIOS write protection](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#dasharo-security-options)
+- [USB stack disable option in setup menu](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#usb-configuration)
+- [Network stack disable option in setup menu](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#networking-options)
+
+### Known issues
+
+- [Display Port has trouble working with certain monitors](https://github.com/Dasharo/dasharo-issues/issues/1015)
+- [STB002.001 encounters unlisted errors](https://github.com/Dasharo/dasharo-issues/issues/1013)
 
-<!--Empty pixel to avoid orphaned pages when overview is hidden-->
-[![empty-pixel](../../images/empty_pixel.png)](overview.md)
+### Binaries
+
+[protectli_vault_adl_v0.9.0.rom][protectli_vault_adl_v0.9.0.rom_file]{.md-button}
+[sha256][protectli_vault_adl_v0.9.0.rom_hash]{.md-button}
+[sha256.sig][protectli_vault_adl_v0.9.0.rom_sig]{.md-button}
```

Comment:
I got 404 error.

Reply:
Are the binaries published yet? There were no binaries for protectli adl available at the moment of adding these links so I have been getting 404 too and that should be expected until they are released, right?

Reply:
Links are correct, the binaries are just not published yet.

Reply:
The bianries are not published yet, thus 404. Although thebinaries names are incorrect. It should be protectli_vp66xx_v0.9.0

Reply:
@philipandag how did you came up with these names? These are certainly not the ones that are generated by Dasharo templates.

Reply:
That what was generated by `dasharo-templates/newsletter/run.sh` using the `.yaml` files from https://git.3mdeb.com/dasharo/dasharo/pulls/105. There is no creative invention of mine here

Reply:
Then the content was outdated and not synchronized after changes in dasharo/dasharo repo

Reply:
Anyways, I have pushed a synced version with fixed template metadata

Reply:
I think i forgot to push the changes yesterday... Sorry and thanks for correcting it.